src/v2/Apps/Partner/__tests__/PartnerApp.jest.tsx

```diff
@@ -41,10 +41,23 @@ const { getWrapper } = setupTestWrapper<PartnerApp_Test_Query>({
 
 describe("PartnerApp", () => {
   it("displays navigation tabs for the partner page", () => {
-    const wrapper = getWrapper()
+    const wrapper = getWrapper({
+      Partner: () => ({
+        isNonSubscriber: false,
+      }),
+    })
     expect(wrapper.find("NavigationTabs").length).toBe(1)
   })
 
+  it("doesn't display navigation tabs if non-subscriber profile", () => {
```

Comment:
Here too - would red much better in my opinion if we could say something like `does not display nav tabs for limited profile` or something along those lines.

Reply:
Fixed